compendium/test_bundles/ManagedServiceAndFactoryBundle/src/ManagedServiceImpl.cpp

```diff
@@ -0,0 +1,39 @@
+#include "ManagedServiceImpl.hpp"
+
+#include <iostream>
+#include <string>
+
+namespace cppmicroservices {
+namespace service {
+namespace cm {
+namespace test {
+
+TestManagedServiceImpl::TestManagedServiceImpl()
+    : m_counter{0} {
+}
+
+TestManagedServiceImpl::~TestManagedServiceImpl() = default;
+
+void TestManagedServiceImpl::Updated(AnyMap const& properties) {
+    std::lock_guard<std::mutex> lk(m_counterMtx);
+    if (properties.empty()) {
+        // Usually corresponds to the configuration being removed
+        std::cout << "    Updated called with empty properties" << std::endl;
```

Comment:
Should we be outputting things during tests? Doesn't this clutter the logs?

Reply:
These aren't necessary and should be removed. Thanks!